src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -1147,20 +962,23 @@ internal static SqlConnectionIPAddressPreference ConvertToIPAddressPreference(st
             }
         }
         #endregion
-
-        internal static bool IsValidCertificateValue(string value)
-        {
-            return string.IsNullOrEmpty(value)
-                || value.StartsWith("subject:", StringComparison.OrdinalIgnoreCase)
-                || value.StartsWith("sha1:", StringComparison.OrdinalIgnoreCase);
-        }
     }
 
     internal static class DbConnectionStringDefaults
     {
-        // all
-        //        internal const string NamedConnection           = "";
         private const string _emptyString = "";
+        internal const ApplicationIntent ApplicationIntent = Microsoft.Data.SqlClient.ApplicationIntent.ReadWrite;
+        internal const string ApplicationName =
+#if NETFRAMEWORK
+            "Framework Microsoft SqlClient Data Provider";
+#else
+            "Core Microsoft SqlClient Data Provider";
+#endif
+        internal const string AttachDBFilename = _emptyString;
+        internal const int CommandTimeout = 30;
+        internal const int ConnectTimeout = 15;
+
+#if NETFRAMEWORK
         // Odbc
         internal const string Driver = _emptyString;
         internal const string Dsn = _emptyString;
```

Comment:
We create two constants with empty string values but I can't think of any good reason to do that instead of just using empty string. These lines are just confusing, weird old code.

Reply:
I removed null-referenced old constants. Pruning the stale keywords could be done in a separate PR.

Reply:
excellent, thanks :)